master/internal/config/resource_manager_config.go

```diff
@@ -303,11 +303,9 @@ func (k *KubernetesResourceManagerConfig) UnmarshalJSON(data []byte) error {
 func (k KubernetesResourceManagerConfig) Validate() []error {
 	var checkSlotType error
 	switch k.SlotType {
-	case device.CPU, device.CUDA:
-	case device.ROCM:
-		checkSlotType = errors.Errorf("rocm slot_type is not supported yet on k8s")
+	case device.CPU, device.CUDA, device.ROCM:
```

Comment:
Does the code detect when the user inputs `gpu` and switches it to `cuda`? This makes it seem like `gpu` isn't actually accepted for `slotType`?

Reply:
Yeah we switch it here

https://github.com/determined-ai/determined/blob/80822ebe40e3516ca987872429bb6cac97de7200/master/internal/config/resource_manager_config.go#L297

Reply:
Thanks!!